EosioSwiftVault/EosioVault.swift

```diff
@@ -90,13 +90,18 @@ public final class EosioVault {
     ///
     /// - Parameters:
     ///   - secureEnclave: Generate this key in Secure Enclave?
+    ///   - protection: Accessibility defaults to whenUnlockedThisDeviceOnly.
     ///   - bioFactor: The `BioFactor` for this key.
     ///   - metadata: Any metadata to associate with this key.
+
     /// - Returns: The new key as a VaultKey.
     /// - Throws: If a new key cannot be created.
     /// - Important: Metadata must follow the rules for JSONSerialization.
     /// - SeeAlso: https://developer.apple.com/documentation/foundation/jsonserialization
-    public func newVaultKey(secureEnclave: Bool, bioFactor: BioFactor = .none, metadata: [String: Any]? = nil) throws -> EosioVault.VaultKey {
+    public func newVaultKey(secureEnclave: Bool,
+                            protection: Keychain.AccessibleProtection = .whenUnlockedThisDeviceOnly,
```

Comment:
@Todd-Bowden Should protection be exposed on `newSecureEnclaveKey` as well?

Reply:
Yes